<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventana_principal</class>
 <widget class="QWidget" name="ventana_principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Peliculas argentinas</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(246, 245, 244);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QFrame" name="busquedas">
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="titulo_busqueda_pelicula">
        <property name="text">
         <string>Buscar pelicula</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="busqueda_pelicula">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(192, 191, 188);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Plain</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="texto_busqueda_pelicula">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="placeholderText">
            <string>nombre de pelicula</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="boton_buscar_pelicula">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Buscar - Recargar</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="titulo_busqueda_actores">
        <property name="text">
         <string>Buscar por actores</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="busqueda_actores">
        <property name="styleSheet">
         <string notr="true">
background-color: rgb(192, 191, 188);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Plain</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="texto_nombre_primer_actor">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="placeholderText">
            <string>nombre primer actor</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="texto_nombre_segundo_actor">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="placeholderText">
            <string>nombre segundo actor</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="boton_buscar_actores">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Buscar</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="table_peliculas">
        <row>
         <property name="text">
          <string>Peliculas</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>Peliculas</string>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string>otra</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>BuscadorPeliculas__buscar_por_titulo()</slot>
 </slots>
</ui>
